examples/dotNetClient/DotNetYuduAPIExample/Program.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Globalization;
+using System.Security.Cryptography;
+using System.Text;
+using RestSharp;
+
+namespace DotNetYuduAPIExample
+{
+    internal class Program
+    {
+        private const string SecretKey = "Please contact Yudu to obtain this shared secret.";
+        private const string ApiKey = "Please contact Yudu to obtain this API key.";
```

Comment:
Yeah, was just copying https://github.com/yudugit/rest-api-documentation/blob/master/examples/phpClient/examples.php :p

How about "Please contact Yudu if you don't already have this..."?